src/app/components/modal/modal.component.spec.ts

```diff
@@ -19,7 +21,8 @@ describe('ModalComponent', () => {
     fixture.detectChanges();
   });
 
-  it('should create', () => {
+  //currently failing
```

Comment:
Perhaps we could include a more detailed description of why, if the test is failing, we're still including it in the codebase. Is there some functionality that it should be testing?